index.html

```diff
@@ -198,6 +198,49 @@ <h2 class="seventh-sheet__contact-me">Contact me</h2>
     </form>
       <div class="end-line"><img src="./images/end-line.png" alt="end-line"></div>
   </section>
+  
+<div class="detail" id="detail">
+  <div class="detail-content" id="detailContent">
+    <div class="title-model">
+      <h3 class="detail-title" id="detailTitle">Profesional Art Printing Data</h3>
+      <a href="javascript:void(0)" class="btn-close" onclick="closeDetail()">&times;</a>
+    </div>
+    <div class="details__area-frame">
+      <h4 class="fifth-sheet__area-frame-canopy">Facebook</h4>
+      <img class="fifth-sheet__area-frame-dot" src="./images/dot.png" alt="dot">
+      <h4 class="fifth-sheet__area-frame-BackEndDev">Lead Developer</h4>
+      <img class="fifth-sheet__area-frame-dot" src="./images/dot.png" alt="dot">
+      <h5 class="fifth-sheet__area-frame-BackEndDev">2015</h5>
+    </div>
+    <img src="./images/MultiSotry.png" class="detail-image" id="detailImage" alt="Web pag image">
+    <div class="dereca">
+      <p class="detail-description" id="detailDescription">
+        A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+        industry's standard
+      </p>
+      <div class="detail-user">
+        <div class="detail-technologies">
+          <div class="footer__languages-html footer__languages-text">html</div>
+          <div class="footer__languages-css footer__languages-text">Ruby on rails</div>
+          <div class="footer__languages-css footer__languages-text">css</div>
+          <div class="footer__languages-javascript footer__languages-text">javaScript</div>
+        </div>
+        <div class="detail-buttons" id="detail-buttons">
+          <a href="#" id="btnDetailLive" target="_blank" class="btn-detail button-see-project">See Live<img src="./icons/icon-export.png" alt="icone-export"
```

Comment:
- [ ] As requested by previous reviewers, some more styling could be added to ensure your design is as close to the Figma design.

Your template (Mobile view)
![image](https://user-images.githubusercontent.com/81932478/169661106-ad95746d-24f8-4a5a-a434-1e7c18481ff9.png)

Figma design (Mobile view)
![image](https://user-images.githubusercontent.com/81932478/169661187-3e04c8ef-fe97-4ee3-b9ea-37325db1d28f.png)

Your Desktop design
![image](https://user-images.githubusercontent.com/81932478/169661213-a8b2f78c-f597-423c-a513-a1d62de99699.png)

Figma Desktop design
![image](https://user-images.githubusercontent.com/81932478/169661250-6fab72da-fecb-4385-95fb-382faecf341c.png)

Clearly, the two designs are off by a lot in terms of margin, padding, and positioning.

1. ### * You still have one extra review left, make the required changes, reach out to coding partners for help. You can do this!